dataland-frontend/tests/e2e/utils/EuTaxonomyFinancialsUpload.ts

```diff
@@ -245,6 +245,7 @@ export function gotoEditForm(companyId: string, expectIncludedFile: boolean): vo
  * @param companyInformation Company information to be used for the company upload
  * @param testData EU Taxonomy dataset for financial companies to be uploaded
  * @param beforeFormFill is performed before filling the fields of the upload form
+ * @param formFillSteps Steps involved to fill data of the upload form
```

Comment:
With this new flexibility of filling the whole form added to the function, we can replace also the corresponding function for the non-financial service data upload, i.e. replace `uploadCompanyViaApiAndEuTaxonomyDataForNonFinancialsViaForm`  by this one

Reply:
Done.